prefs.js

```diff
@@ -294,6 +294,30 @@ function layouts_settings_tab(notebook) {
 
     parent_vbox.add(gaphbox);
 
+    const sgaphbox = new Gtk.HBox();
+    const sgap = Gtk.Scale.new_with_range(Gtk.Orientation.HORIZONTAL, 0, 50, 1);
```

Comment:
For large screens like mine, it'll be better to increase the maximum the edge gap size:
```suggestion
    const sgap = Gtk.Scale.new_with_range(Gtk.Orientation.HORIZONTAL, 0, 200, 1);
```

Reply:
I get that. I didn't change it since most users will use this option to make the screen gap smaller

Reply:
@nununoisy not sure what you mean, the default is 0 gap (afaik) so making the optional maximum size larger should be fine for whoever wants it. Btw, here's a screenshot with the workspace screen gap size set to 200... which actually is just perfect for my 49". 
![MVIMG_20190726_120517](https://user-images.githubusercontent.com/595843/61975490-1eec3180-af9e-11e9-8efa-10c94bfa5ae1.jpg)

Reply:
I wasn't even aware of that use case. But I can add it.

Just a quick question, are you also experiencing the positioning bug I described above?

Reply:
@nununoisy sadly I am... as well as other odd glitches (ill mention in the main comments).